geowebcache/core/src/main/java/org/geowebcache/config/XMLConfiguration.java

```diff
@@ -431,7 +431,7 @@ public static XStream getConfiguredXStreamWithContext(XStream xs, WebApplication
             xs.allowTypesByWildcard(new String[]{"org.geowebcache.**"});
         }
         
-        xs.setMode(XStream.NO_REFERENCES);
+        xs.setMode(XStream.ID_REFERENCES);
```

Comment:
Checking what ID_REFERNECE is ... see - http://x-stream.github.io/graphs.html

So this helps with readability when writing by hand - can we confirm that this change will not immediately invalidate existing configurations?